keyboards/hecomi/alpha/rules.mk

```diff
@@ -72,11 +72,12 @@ SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = no            # USB Nkey Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
-RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
 MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
 SPLIT_KEYBOARD = yes
+RGBLIGHT_SPLIT=yes
```

Comment:
This doesn't do anything.  It would need to be in the config.h, but I don't think you need it, since you have the led num split defined.

```suggestion
```

Reply:
I defined RGBLED_SPLIT in config.h, last line.

Reply:
Yup, I saw that. 

However, this line should still be removed.  It doesn't actually do anything, as it's not parsed anywhere.   So, it's an extra, non-operational line, and should be removed.

Reply:
Really? OK, I understand and merge.

Reply:
Yup. I ... actually had to double check to make sure. And yeah, doesn't do anything here, since there is no handling for it.  

And thanks!